cedexis/terraform-provider-citrixitm
# Handle DNS app deleted via Portal

# Summary

When a DNS app managed by Terraform is manually deleted via the Portal, Terraform should recreate it. See #7 

#  Testing

Testing should involve:

1. Create a DNS app via Terraform.
2. Delete the app via the Portal.
3. Run `terraform apply` again and see that Terraform offers to recreate it. A message in the Terraform log file (if logging is enabled) will say something like:

> 2019-02-25T18:31:24.386-0800 [DEBUG] plugin.terraform-provider-citrixitm: 2019/02/25 18:31:24 The app is disabled. This likely means that it was deleted outside of Terraform.

This PR also includes some changes to account for naming changes in the go-itm v0.0.1 dependency.
